docs/bokeh/source/docs/user_guide/server/deploy.rst

```diff
@@ -17,6 +17,52 @@ requirements and the expected number of users.
 However, if you have authentication, scaling, or uptime requirements, you'll
 have to consider more sophisticated deployment configurations.
 
+Integrating Bokeh server into other web services
+------------------------------------------------
+
+Bokeh server commonly acts as a solution for constructing plots and
+dashboards and seamlessly embedding them within a larger parent application.
+To accomplish this, the ``bokeh.embed`` module offers the
+:func:`~bokeh.embed.server_document` and :func:`~bokeh.embed.server_session`
+methods. See :ref:`ug_output_embed_apps` for a detailed discussion of their
+usage, with examples.
+
+If the parent service you wish to integrate with is not Python-based, you can
+still integrate with Bokeh through the ``server_document`` / ``server_session``
+methods, but you will need to do so by calling out to a small, long-lived
+Python script that returns the text contents of those methods via any standard
+form of IPC. Alternatively, you can copy the static contents of the autoloader
+script into your parent app directly as a string, and change the script tag's
+id attribute yourself. However, this approach is not recommended, as it will
+require manually validating the script text stored in your parent app as you
+upgrade across Bokeh versions, and may break entirely across major versions if
+the autoloader workflow is changed.
+
+You will need to add the public hostname of your Bokeh server instance to the
```

Comment:
I think this sentence paragraph needs some motivation, i.e. should start like "In order to <accomplish the thing>, you will need to ..." what is "the thing" in this case?

Reply:
Maybe actually a comment for the top of the section to. IIRC this section is really to offer advice for the needs *specifically* around CSP configuration, maybe that could be stated from the start.